modules/qtwidgets/src/properties/propertywidgetqt.cpp

```diff
@@ -446,8 +446,29 @@ bool PropertyWidgetQt::event(QEvent* event) {
     if (event->type() == QEvent::ToolTip && property_) {
         event->accept();
         auto helpEvent = static_cast<QHelpEvent*>(event);
-        const std::string info = property_->getDescription();
-        QToolTip::showText(helpEvent->globalPos(), utilqt::toQString(info));
+
+        Document desc{};
+        auto html = desc.append("html");
+        html.append("head").append("style", R"(
+            div.name {
```

Comment:
can't this be in the global qss file?

Reply:
How? and it might not be global state to begin with, different tooltips might want different styles to some degreee

Reply:
That is a good point regarding different styles. Also tooltips do not allow forsetting a global stylesheet similar to QTextBrowser.